### ISSUE TITLE
stackoverflow is not a support platform

### ISSUE THREAD

**Issue by trendzetter** [ng-bootstrap/ng-bootstrap#2030]
It does not allow questions without working for them first (required reputation point).

Your "getting help" suggest we can post questions there but we can't.

**Comment by pkozlowski-opensource** [comment]
It is definitively not a perfect support platform, but this is the one we've chosen to use as we believe that this is the best option. You just need 1 point of reputation to ask question: https://stackoverflow.com/help/privileges/create-posts

Again, I hear what you are saying but given all the pros & cons we are going to stick with SO for now.

**Comment by trendzetter** [comment]
it says 15 points to open a question

**Comment by trendzetter** [comment]
I am unable to use the support channel

**Comment by pkozlowski-opensource** [comment]
> it says 15 points to open a question

To be clear - it says _1_ point here: https://stackoverflow.com/help/privileges/create-posts

> Privilege Type: Creation Privilege
> Awarded At: 1 Reputation
> What is asking and answering questions?

**Comment by trendzetter** [comment]
I tried to ask some clarification in the comments but it wasn't allowed. Then I created a question. I found a solution myself, I removed the question but then I lost all the point I had. I reposted the question, someone voted it down but posted a retarded answer with it so I couldn't delete the cloned question anymore. Then I undeleted my original question with the points but it was removed as a duplicate and my carefully crafted answer too deleted by moderators.
I don't like stack overflow. It's the internet at it's worst. It's giving me bad mood.
